image/cli/install/install-yq.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Install yq CLI
+set -e
+
+curl -L  "https://github.com/mikefarah/yq/releases/download/v4.35.1/yq_linux_amd64"  > yq
```

Comment:
The previous yq was this one https://pypi.org/project/yq/ (https://github.com/kislyuk/yq) which is different to https://github.com/mikefarah/yq is there a reason for the change? I believe we are using the original yq in places

Reply:
I cannot find any references to this and that is why I removed it completely:

```bash
# Install yq
# ----------
function install_yq() {
  python -m pip install yq || exit 1
}
```

I feel the version of `yq` from `mikefarah` is better than the Python one. Also that is the one we used to develop the backup and restore scripts.

Reply:
@terenceq it is being used in a few places in the gitops branch like here https://github.com/ibm-mas/cli/blob/gitops/image/cli/mascli/functions/gitops_init_suite#L323 and also the prereq installed jq that comes with it. We choose that as that is what is installed.

Reply:
@whitfiea ok - I put the `mikefarah` version under `/opt/mikefarah/bin` and I'll introduce some logic in the backup/restore scripts that will allow setting the path to the yq binary in a later PR.